indico/modules/events/abstracts/models/comments.py

```diff
@@ -60,7 +67,7 @@ class AbstractComment(db.Model):
         index=True,
         nullable=False
     )
-    text = db.Column(
+    _text = db.Column(
```

Comment:
`'text'` as the first argument so it doesn't expect the column to be named `_text`